.github/workflows/main.yml

```diff
@@ -0,0 +1,28 @@
+name: Continuous integration
+
+on: [pull_request]
+
+jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+          cache: 'npm'
+
+      - name: Install
+        run: npm ci
+
+      - name: Code style
+        run: npm run lint
+
+      - name: UI E2E Tests
+        uses: cypress-io/github-action@v2
+        with:
+          browser: chrome
+          headless: true
```

Comment:
EOF.
Add a blank line to end of the file to remove this warning.
![image](https://user-images.githubusercontent.com/10209180/169912507-c75cdc43-3194-46a9-a757-eae924afd9a3.png)

You can configure visual studio code to fix this automatically:
https://stackoverflow.com/a/44704969